### DISCUSSION TITLE
Have arrow elbows always go from low->high

### DISCUSSION THREAD

**Discussion by nicpottier** [nyaruka/floweditor#1144] [Ideas]
Just recording a random thought.

I think we already mess with the elbow length for arrows coming out a ruleset, but I wonder if we are quite doing the right thing. I think we are trying to have the center be the "lowest" right now which works well when the nodes are centered below the item, but I'm not sure that is the typical case.

Wonder if this is something we should try to do dynamically? Seems it would really reduce the perceived complexity if we could cause these not to cross.

Two examples (bad and good) attached.

<img width="698" alt="Screen Shot 2020-04-20 at 11 33 03 AM" src="https://user-images.githubusercontent.com/168067/79786749-ccf0f600-82fa-11ea-8f66-9efe5fe4074f.png">
<img width="698" alt="Screen Shot 2020-04-20 at 11 33 13 AM" src="https://user-images.githubusercontent.com/168067/79786765-d0847d00-82fa-11ea-8df9-14178ce38065.png">

Related to #829 though this is more 1-1 case.

**Comment by ericnewcomer** [comment]
Totally agree the overlap just totally turns my brain off in trying to make sense of what is happening there. It would be fun to experiment more with this to see if we can avoid it more consistently. As is always the case with this, overfitting one scenario / layout pattern can also be problematic in other common layouts.

I think this is worth another look.
